apps-we-use/openvpn.md

```diff
@@ -0,0 +1,30 @@
+# OpenVPN
+
+We use OpenVPN to securely tunnel internet traffic when connecting to
+our projects or internal services.
+
+## Installation
+
+#### MacOS
+  * [Tunnelblick] desktop application.
+
+#### Ubuntu
+  * OpenVPN and network plugin, `apt install network-manager-openvpn-gnome`
```

Comment:
Why not just `apt install openvpn`?

Reply:
So you have it in the network manager gui. With the latest versions of Ubuntu there should not be any issues with loading `ovpn` files into the network manager.

That cli usage is not installing a configuration though. It is just running openvpn with the file, not a sustainable approach, compared to clicking on the menu in the statusbar. It could be added as a note to test the configuration file if having issue with network manager.

Reply:
> not a sustainable approach

It's super easy just to reverse search for the command and retype this (or just rewrite the basic command). It's not cumbersome in my opinion, and it's the method I've used over the past year. With this approach in the documentation, you can use it on Mac or any Unix-like system. What's the harm in adding an additional (simple) approach that works on more systems, especially when the approach is just a command?

If someone has a problem getting the VPN setup with a network GUI, or maybe they're using Fedora or whatever, they can resort to using the command. I see added benefit there.

Reply:
Added command line details